python/dgl/transforms/functional.py

```diff
@@ -68,13 +68,11 @@
     'to_block',
     'to_simple',
     'to_simple_graph',
-    'as_immutable_graph',
     'sort_csr_by_tag',
     'sort_csc_by_tag',
     'metis_partition_assignment',
     'partition_graph_with_halo',
     'metis_partition',
-    'as_heterograph',
```

Comment:
Again, describe these deprecated APIs in the PR description and link appropriate tracker items.

Reply:
revert